client/modules/IDE/components/Editor.js

```diff
@@ -79,11 +98,26 @@ class Editor extends React.Component {
   _cm: CodeMirror.Editor
 
   render() {
-    return <div ref="container" className="editor-holder" tabIndex="0" title="code editor" role="region"></div>;
+    return (
+      <div>
+        <div ref="container" className="editor-holder" tabIndex="0" title="code editor" role="main">
+        </div>
+        <EditorAccessibility
+          lintMessages={this.props.lintMessages}
+          lineNo={this.props.lineNo}
```

Comment:
can we call this lineNumber instead of lineNo? this applies for every instance of lineNo